ethers-core/src/types/transaction/response.rs

```diff
@@ -1071,4 +1069,24 @@ mod tests {
         decoded_tx.hash = decoded_tx.hash();
         dbg!(&decoded_tx);
```

Comment:
let's remove this as well :)

Reply:
removed & replaced with an assert for the hash!